<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>15</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*메일 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>45</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*이름 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>75</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*전화 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>105</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*주소 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SNS : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>164</y>
     <width>193</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>* 는 필수 입력사항입니다.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>186</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>186</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mail">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>12</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>42</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_telno">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>70</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_addr">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>100</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sns">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>132</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
